static/src/pages/docs/[...page].astro

```diff
@@ -19,7 +19,7 @@ const currentPage = page.currentPage ?? 1;
 
 const categorizedDocs = {
   gettingStarted: page.data.filter((doc) =>
-    ['postgres', 'monitoring', 'pgvector'].includes(doc.slug)
+    ['getting-started', 'projects', 'monitoring', 'pgvector'].includes(doc.slug)
```

Comment:
Rather than this can we add category to the front matter and then filter based upon `category == 'gettingStartinged'` or something